tutorial/westeros/westeros_addon_technologies.ipynb

```diff
@@ -0,0 +1,607 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Westeros Tutorial\n",
+    "## Adding additional technology functionalities: Introducing `addon` technologies\n",
+    "\n",
+    "This tutorial is concerned with is looking at how to add additional features, known as `addon` technologies such as carbon-capture-and-storage (CCS) retrofits, passout-turbines (for optional heat cogeneration) or cooling technologies, to existing technologies.\n",
+    "\n",
+    "There are several ways in which to tackle this issue.  Lets take for example our `coal_ppl`.  All of the above mentioned additional features could be implemented by introducing different modes for the `coal_ppl`.  For example, heat cogeneration could be implemented as a separate `mode` of the `coal_ppl`, where instead of just generating electricity, heat can also be produced at the cost of reducing the amount of electricity generated. Another approach would make use of the generic `relations`, therefore linking the a newly added technology representing the passout-turbine with the activity of the `coal_ppl`.  Both of these approaches have some downsides. Using a separate `mode` will not permit explicitly modelling investment costs and lifetimes associated with the asset being added to the `coal_ppl`.  Generic relations are very flexible, but if too many of these exist, then the modell quickly becomes very hard to understand.  Therefore, MESSAGEix offers an explicit `addon` formulation.\n",
+    "\n",
+    "What this means, is that the additional technology options are explicitly modelled as separate technologies, classified as `addon` technologies, linked to the activity of the technology to which they serve as additional configuration options, the parent technology, through specifying a `addon_conversion` factor.  The activity of the `addon` technology can further be restricted to a minimum or maximum share of the activity of the parent technology."
+   ]
+  },
```

Comment:
Mentio "requirements" for running this tutorial, like running "baseline" or any other tutorial beforehand. (please refer to other tutorials to see examples).

Reply:
done